src/main/resources/net/uaznia/lukanus/hudson/plugins/gitparameter/Messages_fr.properties

```diff
@@ -0,0 +1,10 @@
+GitParameterDefinition.DisplayName=Git Parameter
+GitParameterDefinition.noRepositoryConfigured=Aucun dépôt git congiguré parmi les configurations scm
```

Comment:
If you use diacritical sign you must ust utf encoding
eg https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/Messages_fr.properties 
https://websitebuilders.com/tools/html-codes/french/